core/storage/classifier/gae_models.py

```diff
@@ -175,13 +183,35 @@ def create(
             interaction_id=interaction_id,
             exp_id=exp_id,
             exp_version=exp_version,
-            state_name=state_name, status=status, training_data=training_data
+            next_scheduled_check_time=next_scheduled_check_time,
+            state_name=state_name, status=status,
+            training_data=training_data,
             )
```

Comment:
Either deindent by 4, or move to end of previous line (and elide the last comma).